code/game/objects/items.dm

```diff
@@ -1,835 +1,860 @@
-GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/effects/fire.dmi', "fire"))

```

Comment:
was it necessary to change the line endings of the entire file?

Reply:
fucking web edits hold on